osu.Game/Migrations/20181007180454_StandardizePaths.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+using System.IO;
+
+namespace osu.Game.Migrations
+{
+    public partial class StandardizePaths : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            string sanitized = Path.DirectorySeparatorChar.ToString();
```

Comment:
This should be "\\" as it will fail to properly migrate, e.g. `AudioFilename = folder\track.mp3` on unix for maps imported _prior_ to this PR.

Reply:
Fixed.